Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml

```diff
@@ -8,6 +8,19 @@
       role: JobPassenger
       time: 36000 #10 hrs, silly reward for people who play passenger a lot
 
+# Head
+- type: loadout
+  id: PassengerHead
+  equipment: Cone
+  effects:
+  - !type:GroupLoadoutEffect
+    proto: GreyTider
+
+- type: startingGear
+  id: Cone
+  equipment:
+    head: ClothingHeadHatCone
+
```

Comment:
Timegating this one is silly.

Reply:
Thought it seemed fitting as the others had a time limit and are found in maints.